internal/test/e2e/e2e_test.go

```diff
@@ -83,8 +85,15 @@ func getSpec(t *testing.T, o interface{}) interface{} {
 
 // Starts a Kind cluster on the local machine, exposing port 2379 accepting ETCD connections.
 func startKind(t *testing.T, ctx context.Context) (*cluster.Context, error) {
-	t.Log("Starting Kind cluster")
-	kind := cluster.NewContext("etcd-e2e")
+	clusters, err := cluster.List()
+	for _, c := range clusters {
+		if c.Name() == *fKindClusterName {
+			t.Log("Found existing Kind cluster")
```

Comment:
are we confident that reusing a cluster will not have any side effects that would cause tests to run inconsistently?